pkgs/applications/misc/libosmocore/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, pcsclite, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  name = "libosmocore-${version}";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "osmocom";
+    repo = "libosmocore";
+    rev = "8649d57f507d359c99a89654aac7e19ce22db282";
+    sha256 = "08mcpy9ljwb1i3l4cmlwn024q2psk5gg9f0ylgh99hy1ffx0n7am";
+  };
+
+  buildInputs = [
+    autoconf automake libtool pcsclite pkgconfig
+  ];
+
+  preConfigure = ''
+    autoreconf -i -f
```

Comment:
This should be replaced by using `autoreconfHook`, making `autoconf automake libtool` unnecessary in buildInputs, too.